packages/@expo/config-plugins/src/plugins/withIosBaseMods.ts

```diff
@@ -85,6 +85,19 @@ const defaultProviders = {
       await writeFile(filePath, contents);
     },
   }),
+  // Append a rule to supply BridgingHeader data to mods on `mods.ios.bridgingHeader`
+  bridgingHeader: provider<Paths.BridgingHeaderProjectFile>({
+    getFilePath({ modRequest: { projectRoot } }) {
+      // TODO: Get application BridgingHeader file from pbxproj.
+      return Paths.getBridgingHeaderFilePath(projectRoot);
```

Comment:
The bridging header should be retrieved correctly by following the pbxproj like the info.plist does.